web/styles/dark_theme.css

```diff
@@ -38,7 +38,7 @@
             color: hsl(236deg 33% 90%/50%);
         }
 
-        & li.unmuted_topic {
+        & li.unmuted_or_followed_topic {
```

Comment:
Rather than update and carry forward this duplicate selector, why not set up a CSS variable for the color, and then just specify the different one in the dark-theme area of `zulip.css`?

Reply:
Yeah, we can do that. But I think we should keep the CSS related to `left_sidebar` in `left_sidebar.css` itself.

The `zulip.css` file is already 3200+ lines.

right?

Reply:
Well, that's where we're keeping color variables. And that's all I'm suggesting you add there, is the variable declaration--not the actual selectors.

Reply:
Done. First commit.